pulpcore/cli/file/locale/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,146 @@
+msgid "file ACS"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:21
+msgid "file ACSes"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:38 pulpcore/cli/file/context.py:39
+msgid "file content"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:47
+msgid "file distribution"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:48
+msgid "file distributions"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:68
+msgid "file publication"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:69
+msgid "file publications"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:86
+msgid "file remote"
+msgstr ""
+
+#: pulpcore/cli/file/context.py:87
+msgid "file remotes"
+msgstr ""
+
+#: pulpcore/cli/file/distribution.py:32
+msgid ""
+"Repository to be used for auto-publishing. When set, this will unset the "
+"'publication'. Specified as '[[<plugin>:]<type>:]<name>' or as href."
+msgstr ""
+
+#: pulpcore/cli/file/distribution.py:63
+msgid ""
+"Publication to be served. This will unset the 'repository' and disable auto "
+"publish."
+msgstr ""
+
+#: pulpcore/cli/file/publication.py:50
+msgid "a repository version number, leave blank for latest"
+msgstr ""
+"Versionsnummer des Repositories, leer lassen um die letzte zu verwenden"
+
+#: pulpcore/cli/file/publication.py:54
+msgid ""
+"Filename to use for manifest file containing metadata for all the files."
+msgstr ""
+
+#: pulpcore/cli/file/repository.py:56
+msgid ""
+"Remote used for synching in the form '[[<plugin>:]<resource_type>:]<name>' "
+"or by href."
+msgstr ""
+
+#: pulpcore/cli/file/repository.py:80
+#, python-brace-format
+msgid "Validation of '{parameter}' failed: {error}"
+msgstr ""
+
+#: pulpcore/cli/file/repository.py:135 pulpcore/cli/file/repository.py:279
+msgid ""
+"JSON string with a list of objects to add to the repository.\n"
+"    Each object must contain the following keys: \"sha256\", \"relative_path"
+"\".\n"
+"    The argument prefixed with the '@' can be the path to a JSON file with a "
+"list of objects."
+msgstr ""
+
+#: pulpcore/cli/file/repository.py:144 pulpcore/cli/file/repository.py:290
+msgid ""
+"JSON string with a list of objects to remove from the repository.\n"
+"    Each object must contain the following keys: \"sha256\", \"relative_path"
+"\".\n"
+"    The argument prefixed with the '@' can be the path to a JSON file with a "
+"list of objects."
+msgstr ""
+
+#: pulpcore/cli/file/repository.py:188
+#, python-brace-format
+msgid ""
+"Repository '{name}' does not have a default remote. Please specify with '--"
+"remote'."
+msgstr ""
+"Repository '{name}' hat keine voreingestellte Gegenstelle. Bitte mit '--"
+"remote' angeben."
+
+#~ msgid "Chunk size in bytes (default is 1 MB)"
+#~ msgstr "Blockgröße in Bytes (Voreinstellung ist 1 MB)"
+
+#, python-brace-format
+#~ msgid "Failed to load content from {json_file}"
+#~ msgstr "Das Laden des Inhalts aus der Datei {json_file} ist fehlgeschlagen"
+
+#~ msgid "Failed to decode JSON"
+#~ msgstr "Die JSON-Dekodierung ist fehlgeschlagen"
```

Comment:
What are these comment-entries?

Reply:
Maybe no longer in the list of translatable strings? IDK. I used poedit to translate.

Reply:
Ah ok - I have a (very) vague memory from a (very) distant past-life of "poedit does this in case the string shows back up again", thanks for jogging my memory!